drivers/bus/Kconfig

```diff
@@ -206,11 +206,12 @@ source "drivers/bus/mhi/Kconfig"
 config VIRTUAL_BUS
        tristate "Software based Virtual Bus"
        help
-         Provides a software bus for virtbus_devices to be added to it
-         and virtbus_drivers to be registered on it.  It matches driver
-         and device based on id and calls the driver's probe routine.
-         One example is the irdma driver needing to connect with various
-         PCI LAN drivers to request resources (queues) to be able to perform
-         its function.
+         Provides a software bus for virtual bus drivers to register
+	 and attach virtual bus devices.  It matches driver and device based
+	 on a unique string and calls the driver's probe routine.
+	 One example is SOF (Sound Open Firmware) clients such as audio cards
+	 or test tools needing to connect with the SOF core. Another example
+         is the irdma driver needing to connect with various PCI LAN drivers
+	 to request resources (queues) to be able to perform its function.
```

Comment:
alignment seems off.

Reply:
hmm this looks fine on my editor. Is it maybe a github display issue?

Reply:
@ranj063 No, it's TABs vs. spaces